import utils.exceptions.svector.InvalidDimensionException

object Main extends App {

  println(SVector.random dot SVector.random)

}
